app-connector-client/server/app.js

```diff
@@ -11,7 +11,8 @@ var connectorModule = require("./routes/connector");
 const events = require("./routes/events")
```

Comment:
you urgently must rebase your branch as this is all conflicting

Reply:
Done